.agents/skills/ansible-galaxy-roles/SKILL.md

```
@@ -0,0 +1,61 @@
+### 1. Role Path Customization (`ansible.cfg`)
+To prevent external dependencies from polluting global system directories or conflicting with localized roles, `ansible.cfg` specifies:
+```ini
+roles_path = roles:common_roles
+```
+This forces all Galaxy and community roles to load or be installed directly within the workspace’s `roles/` (or `common_roles/`) folder.
+
+### 2. Dependency Installation
+To install required external roles defined in `requirements.yml`, execute:
+```bash
+ansible-galaxy role install -r requirements.yml --ignore-errors
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)